mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -122,7 +145,11 @@ CLASS zr net/minecraft/util/math/MathHelper
 	METHOD g isPowerOfTwo (I)Z
 	METHOD h fractionalPart (D)D
 		ARG 0 value
+	METHOD h lerpDegree (FFF)F
```

Comment:
lerpDegree? Sounds odd... can we do better?

Reply:
`lerpAngle`? `interpolateAngle`?

Reply:
Removed. I am not professional on this and will leave it for someone else.